test/cypress/features/common/common.js

```diff
@@ -213,3 +213,25 @@ And(/^I search for account ([^s]+)$/, function (string) {
   cy.wait('@requestAccount');
   cy.wait('@requestDelegate');
 });
+
+Then(/^It should change fee when changing priorities$/, function () {
+  const generateFeePromise = (selector) => new Promise((resolve) => {
+    cy.get(selector).click();
+    cy.get(ss.feeValue).invoke('text').then(resolve);
+  });
+
+  Promise.all([
+    generateFeePromise(ss.lowPriorityFee),
+    generateFeePromise(ss.mediumPriorityFee),
+    generateFeePromise(ss.highPriorityFee),
+  ]).then((values) => {
+      const lowFee = values[0];
+      const mediumFee = values[1];
+      const highFee = values[2];
+      cy.wrap(null).should(() => {
+        expect(lowFee).to.not.equal(mediumFee);
+        expect(lowFee).to.not.equal(highFee);
+        expect(mediumFee).to.not.equal(highFee);
```

Comment:
`expect(mediumFee).not.toEqual(...)` and so on

Reply:
It is using chai's expect that is bundled in cypress, so `toEqual` does not exist.